lib/deploy/eb/platform.rb

```diff
@@ -1,5 +1,11 @@
+require 'open3'
+
 module Eb
   class Platform
+    def self.configured?
+      Open3.popen3("eb list") { |i, o, e, t| t.value }.exitstatus == 0
```

Comment:
Arguments of the block are a little cryptic I must admit.
Also, unused ones could/should be prefixed with an underscore, as per convention.